code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -193,11 +193,13 @@
 		oxyloss = 0
 	return ..()
 
-/mob/living/carbon/human/adjustOxyLoss(var/amount)
+/mob/living/carbon/human/adjustOxyLoss(amount)
 	if(species.flags & NO_BREATHE)
 		oxyloss = 0
 	else
 		amount = amount*species.oxy_mod
+		if(stats.getPerk(PERK_LUNGS_OF_IRON))
```

Comment:
This would make chems like dexalin half as effective.

Is this intended?

Reply:
No. But it does make some weird sense.

User should be notified about this sub effect in perk description.